### PR TITLE
Create Footer 

### PR DESCRIPTION
**Pull request by jyoti-wp**
- [x] Create footer elements
- [x] Add styles

### CODE REVIEW

File: assets/src/scss/components/_client-section.scss [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        margin: 10px 0;
+        font-family: "Noto Serif",serif;
+        font-style: italic;
+        text-align: center;
+        font-size: 24px;
+        margin-left: 15px;
+    }
+
+    p::after {
+        top: 1.6875rem;
```
Comment (imranhsayed) [inline assets/src/scss/components/_client-section.scss:39]: Use `px` everywhere not `rem` for consitency.
Check the `computed` tab to know what `px` value is being applied to the HTML element.

File: assets/src/scss/components/_client-section.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        display: inline-block;
+        width: 6.25rem;
+        height: 0.1875rem;
+        background: #fff;
```
Comment (imranhsayed) [inline assets/src/scss/components/_client-section.scss:43]: Use color variable $white

File: assets/src/scss/components/_client-section.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    }
+
+
+    .image-container img {
+        
+    }
+
+    a{
+        display: block;
+        border: 1px solid #eaeaea;
```
Comment (imranhsayed) [inline assets/src/scss/components/_client-section.scss:62]: Use color variables everywhere

File: assets/src/scss/components/_client-section.scss [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        padding-bottom: 5.125rem;
+        margin-left: 15px;
+        margin-right: 15px;
+
+    }
+
+
+    .image-container img {
+        
+    }
```
Comment (imranhsayed) [inline assets/src/scss/components/_client-section.scss:58]: Why is this empty?  If this is not needed please remove.

File: assets/src/scss/components/_header.scss [hunk 5]
```diff
@@ -1,31 +1,35 @@
...
-
-		@media only screen and ( min-width: $desktop ) {
-			display: block;
-
 		}
 
-		.nav-wrap {
+	.nav-wrap {
 			display: flex;
 		}
```
Comment (imranhsayed) [inline assets/src/scss/components/_header.scss:21]: Check indentation

File: assets/src/scss/components/_header.scss [hunk 6]
```diff
@@ -1,31 +1,35 @@
...
-		a {
-			color: $black;
-		}
+	.nav-list-item {
+
+	}
+
+	.nav-list-item li{
+		
+	}
```
Comment (imranhsayed) [inline assets/src/scss/components/_header.scss:29]: Empty selectors?

File: assets/src/scss/components/home/_contact-us-section.scss [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+    p::after {
+        top: -5px;
+        display: inline-block;
+        width: 100px;
+        height: 3px;
+        background: #26d0ce;
+        background: -webkit-linear-gradient(90deg,#26d0ce,#1a2980);
+        background: -ms-linear-gradient(90deg,#26d0ce,#1a2980);
+        background: linear-gradient(90deg,#26d0ce,#1a2980);
```
Comment (imranhsayed) [inline assets/src/scss/components/home/_contact-us-section.scss:63]: Use Color variables

File: assets/src/scss/components/home/_get-in-touch.scss [hunk 8]
```diff
@@ -0,0 +1,226 @@
...
+        position: relative;
+        margin-bottom: 50px;
+        background: #fafafa;
+        padding: 40px;
+    }
+
+
+    .action-page {
+        
+        @media screen and (min-width: $desktop) {
```
Comment (imranhsayed) [inline assets/src/scss/components/home/_get-in-touch.scss:56]: Can we add some space for readability like so :
` @media screen and ( min-width: $desktop ) {`

File: assets/src/scss/components/home/_get-in-touch.scss [hunk 9]
```diff
@@ -0,0 +1,226 @@
...
+        color: #272727;
+        background: #fff;
+        text-transform: capitalize;
+        width: 243px;
+        height: 26px;
+        font-size: 12px;
+
+        @media screen and (min-width: $tablet) {
+
+        }
```
Comment (imranhsayed) [inline assets/src/scss/components/home/_get-in-touch.scss:79]: ?

File: assets/src/scss/components/home/_get-in-touch.scss [hunk 10]
```diff
@@ -0,0 +1,226 @@
...
+            padding: 20px;
+            margin-left: 10px;
+            border-radius: 35px;
+        }
+
+    }
+
+    .contact-area-form {
+        
+    }
```
Comment (imranhsayed) [inline assets/src/scss/components/home/_get-in-touch.scss:139]: ?

File: assets/src/scss/components/home/_get-in-touch.scss [hunk 11]
```diff
@@ -0,0 +1,226 @@
...
+
+
+
+        // get in touch area section
+
+
```
Comment (imranhsayed) [inline assets/src/scss/components/home/_get-in-touch.scss:145]: Remove extra blank lines

File: assets/src/scss/settings/_colors.scss [hunk 12]
```diff
@@ -6,5 +6,5 @@ $white: #fff;
 $light-gray: #f3f5f6;
 $red: #ff0000;
 
-$teel: #5089a3;
-$footer: #26272c;
+$aqua: #26d0ce;
```
Comment (imranhsayed) [inline assets/src/scss/settings/_colors.scss:9]: Good 👍

File: assets/src/scss/settings/_reset.scss [hunk 13]
```diff
@@ -30,6 +30,9 @@ footer, header, hgroup, menu, nav, section {
 }
 body {
 	line-height: 1;
+	font-family: sans-serif;
+	font-size: 18px/30px Poppins,sans-serif;
```
Comment (imranhsayed) [inline assets/src/scss/settings/_reset.scss:34]: Please remove this font size, you have already added this in `_typography.scss`

File: templates/layouts/components/final-footer.php [hunk 14]
```diff
@@ -0,0 +1,25 @@
...
+            </a>
+            <a href="https://pinterest.com/" class="" target="_blank">
+                <i class="fa fa-pinterest" aria-hidden="true"></i>
+            </a>
+            <a href="https://web.youtube.com/" class="" target="_blank">
+                <i class="fa fa-youtube" aria-hidden="true"></i>
+            </a>
+        </div>
+        <div class="template-text-area">
+            <h5>Templated by <a href="http://www.themeforest.net/user/themeexpo"> ThemeExpo </a>
```
Comment (imranhsayed) [inline templates/layouts/components/final-footer.php:21]: Please change the URL to # and ThemeExpo to `Demo`

File: templates/layouts/components/media-gallery.php [hunk 15]
```diff
@@ -0,0 +1,33 @@
+<div class="media-gallery">
+    <h5 class="text-area">Instagram</h5>
+    <div class="gallery-items-landscape">
+                <div class="gallery-items">
+                    <div class="gallery-items-images">
+                        <img width="150" height="150" src="http://wp.efforttech.net/newwp/sigma/wp-content/uploads/2019/01/image-1-5-150x150.jpg" class="attachment-thumbnail size-thumbnail" alt="" srcset="http://wp.efforttech.net/newwp/sigma/wp-content/uploads/2019/01/image-1-5-150x150.jpg 150w, http://wp.efforttech.net/newwp/sigma/wp-content/uploads/2019/01/image-1-5-300x300.jpg 300w, http://wp.efforttech.net/newwp/sigma/wp-content/uploads/2019/01/image-1-5-100x100.jpg 100w, http://wp.efforttech.net/newwp/sigma/wp-content/uploads/2019/01/image-1-5-60x60.jpg 60w, http://wp.efforttech.net/newwp/sigma/wp-content/uploads/2019/01/image-1-5.jpg 550w" sizes="(max-width: 150px) 100vw, 150px">
+                    </div>
```
Comment (imranhsayed) [inline templates/layouts/components/media-gallery.php:7]: Please download these images and keep it in the images folder and add the image path here.

File: templates/layouts/components/media-gallery.php [hunk 16]
```diff
@@ -0,0 +1,33 @@
...
+                </div>
+                <div class="gallery-items">
+                    <div class="gallery-items-images">
+                    <img width="150" height="150" src="http://wp.efforttech.net/newwp/sigma/wp-content/uploads/2019/01/image-6-2-150x150.jpg" class="attachment-thumbnail size-thumbnail" alt="" srcset="http://wp.efforttech.net/newwp/sigma/wp-content/uploads/2019/01/image-6-2-150x150.jpg 150w, http://wp.efforttech.net/newwp/sigma/wp-content/uploads/2019/01/image-6-2-300x300.jpg 300w, http://wp.efforttech.net/newwp/sigma/wp-content/uploads/2019/01/image-6-2-100x100.jpg 100w, http://wp.efforttech.net/newwp/sigma/wp-content/uploads/2019/01/image-6-2-60x60.jpg 60w, http://wp.efforttech.net/newwp/sigma/wp-content/uploads/2019/01/image-6-2.jpg 550w" sizes="(max-width: 150px) 100vw, 150px">            </div>
+                </div>
```
Comment (imranhsayed) [inline templates/layouts/components/media-gallery.php:29]: Download these images

File: templates/layouts/components/recent-area.php [hunk 17]
```diff
@@ -0,0 +1,47 @@
...
+                <div class="image-area-section">
+                    <div class="image-area">
+                        <img src="http://wp.efforttech.net/newwp/sigma/wp-content/uploads/2019/01/3-60x60.jpg" alt="">
+                    </div>    
+                    <div class="new-trend-area">
+                        <a class="post-title" href="http://wp.efforttech.net/newwp/sigma/2019/01/major-trends-expected-2018-4/">new trends</a>
+                    <div class="post-details">
+                        <a class="post-author" href="http://wp.efforttech.net/newwp/sigma/author/admin/">by admin</a>
+                    <div class="date-area">   
+                        <a class="post-date" href="http://wp.efforttech.net/newwp/sigma/21/01/">21 Jan</a>
```
Comment (imranhsayed) [inline templates/layouts/components/recent-area.php:12]: Change the href to `#`

File: templates/layouts/header.php [hunk 18]
```diff
@@ -1,36 +1,38 @@
...
+				<li class="nav-list-item">
+					<a href="#">Services</a>
+				</li>
+				<li class="nav-list-item">
+				<a href="#">portfolio</a>
+				</li>
+				<li class="nav-list-item">
+				<a href="#">about us</a>
+				</li>
+				<li class="nav-list-item">
```
Comment (imranhsayed) [inline templates/layouts/header.php:20]: Check indentation

File: templates/layouts/map-section.php [hunk 19]
```diff
@@ -0,0 +1,9 @@
+<main>
+    <div class="mapouter">
+        <div class="gmap-canvas">
+            <iframe width="600" height="500" id="gmap_canvas" src="https://maps.google.com/maps?q=university%20of%20san%20francisco&t=&z=13&ie=UTF8&iwloc=&output=embed" frameborder="0" scrolling="no" marginheight="0" marginwidth="0">
+            </iframe>
+            <a href="https://www.whatismyip-address.com/divi-discount/"></a>
```
Comment (imranhsayed) [inline templates/layouts/map-section.php:6]: This a tag is empty , so we can remove it

File: assets/src/scss/components/_client-section.scss (hunk 2 above)
Comment (jyoti-wp) [reply assets/src/scss/components/_client-section.scss:43]: At times its giving an error . the colors were not reflecting. hence used the color direclty

File: templates/layouts/components/media-gallery.php (hunk 15 above)
Comment (jyoti-wp) [reply templates/layouts/components/media-gallery.php:7]: i have downloaded the images but with direct path its not reflecting. ill check again

File: templates/layouts/components/media-gallery.php (hunk 16 above)
Comment (jyoti-wp) [reply templates/layouts/components/media-gallery.php:29]: okay

File: templates/layouts/components/recent-area.php (hunk 17 above)
Comment (jyoti-wp) [reply templates/layouts/components/recent-area.php:12]: okay. All the href?

File: templates/layouts/map-section.php (hunk 19 above)
Comment (jyoti-wp) [reply templates/layouts/map-section.php:6]: okay

Comment (jyoti-wp) [reply templates/layouts/map-section.php:6]: ill get it done

File: assets/src/scss/common/_general.scss [hunk 20]
```diff
@@ -0,0 +1,10 @@
+.wrapper{
+	margin: auto 0;
+	width: 100%;
+	max-width: 1200px;
```
[the remark this answers is not in the document]
Comment (jyoti-wp) [reply assets/src/scss/common/_general.scss:4]: shall i keep the $desktop. coz both are the same width.

File: assets/src/scss/components/_client-section.scss [hunk 21]
```diff
@@ -0,0 +1,65 @@
+section {
+
+
+    .kc-row-container {
+        background-image:url(http://wp.efforttech.net/newwp/sigma/wp-content/uploads/2019/01/bg-6.jpg);
```
[the remark this answers is not in the document]
Comment (jyoti-wp) [reply assets/src/scss/components/_client-section.scss:5]: okay. Ill download all the images today and save them. will change the path.

File: assets/src/scss/common/_general.scss (hunk 20 above)
[the remark this answers is not in the document]
Comment (imranhsayed) [reply assets/src/scss/common/_general.scss:4]: No @jyoti-wp , $desktop variable is for media query breakpoints. Let's not mix that .
Please keep $site_width or $site_max_width

File: assets/src/scss/components/_client-section.scss (hunk 2 above)
Comment (imranhsayed) [reply assets/src/scss/components/_client-section.scss:43]: @jyoti-wp 
If you have created that variable in _colors.scss, You may need to import that file at the top  of this file first

File: templates/layouts/components/media-gallery.php (hunk 15 above)
Comment (imranhsayed) [reply templates/layouts/components/media-gallery.php:7]: It Should if you give the correct path... 
Remember this file is included in index.php so consider you are in root. 
So path should be something like:
`src="assets/images/xyz.png"`

File: templates/layouts/components/recent-area.php (hunk 17 above)
Comment (imranhsayed) [reply templates/layouts/components/recent-area.php:12]: all the href value where its an external link change `href="#"`

**Pull request merged**